### PR TITLE
Velero 1.1 blog post

### PR DESCRIPTION
**Pull request by jonasrosland**
This will add the Velero 1.1 blog post to the site.

Signed-off-by: Jonas Rosland <jrosland@vmware.com>

### CODE REVIEW

File: site/_posts/2019-08-22-announcing-velero-1.1.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Finally, thanks to testing by [Dylan Murray](https://github.com/dymurray) and [Scott Seago](https://github.com/sseago), an issue with running Velero in non-default namespaces was found in our beta version for this release. If you’re running Velero in a namespace other than `velero`, please follow the [upgrade instructions](https://velero.io/docs/v1.1.0/upgrade-to-1.1/).
+
+## Help Us Build the Future
+
+For Velero 1.2, the current plan is to begin implementing CSI snapshot support at a beta level. If accepted, this approach would align Velero with the larger community, and in the future, it would allow Velero to snapshot far more volume providers. We have posted a [design document](https://github.com/heptio/velero/pull/1661) for community review, so please be sure to take a look if this interests you.
+
+We are also working on volume cloning, so that a persistent volume could be snapshotted and then duplicated for use within another namespace in the cluster.
+
+
```
Comment (nrb) [inline site/_posts/2019-08-22-announcing-velero-1.1.md]: I think this blank line can be removed.

**Comment by nrb** [comment]
Generating this locally, I don't see an HTML file being created for this post.

**Comment by jonasrosland** [comment]
If you use `bundle exec jekyll serve --livereload --future` to build it locally, it will build "future" posts as well, and it should show up.

File: site/_posts/2019-08-22-announcing-velero-1.1.md (hunk 1 above)
Comment (jonasrosland) [reply site/_posts/2019-08-22-announcing-velero-1.1.md]: Removed!

**Comment by skriss** [comment]
ah - does it parse the date from the file name?

**Comment by skriss** [comment]
I'll merge this tomorrow after cutting the release.

**Comment by jonasrosland** [comment]
> ah - does it parse the date from the file name?

Yup!

**Comment by nrb** [comment]
Should this update the “Latest Release” link on the homepage, or is that automatic?

**Comment by skriss** [comment]
looks like the "latest release" and "download" links on the main page are static, i'll put a PR up to update them shortly

**Pull request merged**